### ISSUE TITLE
Need secrets.COCOAPODS_TRUNK_TOKEN

### ISSUE THREAD

**Issue by Wei18** [skywinder/ActionSheetPicker-3.0#565]
Hi @skywinder 

I would like to auto bump cocapods spec, but I need your help.
Please add secrets.COCOAPODS_TRUNK_TOKEN for [automated-cocoapod-releases-with-github-actions](https://medium.com/swlh/automated-cocoapod-releases-with-github-actions-8526dd4535c7)

ref:
- https://github.com/skywinder/ActionSheetPicker-3.0/actions/runs/7707696085/job/21005327847
- https://blog.cocoapods.org/CocoaPods-Trunk-RCE/

**Comment by skywinder** [comment]
@Wei18 Hi. Which email should i add to cocoapods for you?

**Comment by Wei18** [comment]
I'm consider using Github Action w/ secrets, but need your permission to add new secret;
therefore, could you add your token (generated from your email).

You can tap the following link:
https://github.com/skywinder/ActionSheetPicker-3.0/settings/secrets/actions/new 

Btw, it's okay to add my email: 41205mw@gmail.com
However that have no help for this issue.

**Comment by skywinder** [comment]
@Wei18 
1. added you to pod owner for managing it.
2. added token, if I got it right (just grep from  `pod trunk me --verbose` output)

**Comment by skywinder** [comment]
ps.


cant open medium article. bit  believe it's this actions https://github.com/marketplace/actions/deploy-to-cocoapod-action

**Comment by wei18** [comment]
Hi @skywinder 

Please help to update secret token then retry failure job again, thanks.

ref:
[!] Authentication token is invalid or unverified. Either verify it with the email that was sent or register a new session.
https://github.com/skywinder/ActionSheetPicker-3.0/actions/runs/11719754388/job/32643942101#step:4:24

**Comment by skywinder** [comment]
@Wei18 hi. Sorry for late reply. Did you fix it already?

**Comment by wei18** [comment]
@skywinder 
No, I don't have permission to access the repository settings to modify secrets.COCOAPODS_TRUNK_TOKEN.
Once you update the value, I can take care of the rest.

**Comment by wei18** [comment]
Resolved! 🎉 I refreshed `COCOAPODS_TRUNK_TOKEN` with a new trunk session token (valid ~4 months, extends on use), and the DeployCocoaPods workflow now publishes successfully — **2.8.0 is live on CocoaPods trunk**. Closing this; if the token expires again I can refresh it the same way.
